arango/collection.py

```diff
@@ -1992,12 +1992,16 @@ def import_bulk(
             IMPORTANT NOTE: this parameter may go through breaking changes
             in the future where the return type may not be a list of result
             objects anymore. Use it at your own risk, and avoid
-            depending on the return value if possible.
+            depending on the return value if possible. Should not be used
+            if **overwrite** is set to true.
         :type batch_size: int
         :return: Result of the bulk import.
         :rtype: dict | list[dict]
         :raise arango.exceptions.DocumentInsertError: If import fails.
         """
+        if overwrite and batch_size:
```

Comment:
I think it's better to do `if overwrite and batch_size is not None` to not have to worry about what happens when `batch_size == 0`